README.rst

```diff
@@ -131,6 +131,8 @@ Maintenance and Support for SDK Major Versions
 
 Boto3 was made generally available on 06/22/2015 and is currently in the full support phase of the availability life cycle.
 
+Boto (pronounced boh-toh) was named after the fresh water dolphin native to the Amazon river.
```

Comment:
Hey Tim, this section of the README is specifically for our maintenance policy. This may fit better either in the first section, or in it's own subheading like "Why is the AWS SDK for Python called Boto?"

Then maybe we add a couple more lines like:
```suggestion
Boto (pronounced boh-toh) was named after the fresh water dolphin native to the Amazon river. The name was chosen by the author of the original Boto library, Mitch Garnaat, as a reference to the company.
```

Reply:
Sounds good, I moved this to the first section and added that sentence. I could add a subheading too if you think we need it.